src/Screens/HomeScreen/HomeScreen.jsx

```diff
@@ -1,5 +1,6 @@
 import { useEffect } from "react";
 import { getPosts } from "../../actions/postsAction";
+import Hamburger from "../../Components/Hamburger/Hamburger";
 import PostContent from "../../Components/PostContent/PostContent";
 import ProfileCard from "../../Components/ProfileCard/ProfileCard";
 import SideBar from "../../Components/SideBar/SideBar";
```

Comment:
you  can try barrel exports (by creating one index file inside components folder )

Reply:
will do it in next PR.